src/main/java/net/querz/nbt/ListTag.java

```diff
@@ -18,6 +18,9 @@ public ListTag() {
 	}
 
 	public ListTag(List<Tag> list, byte type) {
+		Objects.requireNonNull(list);
+		list.forEach(Objects::requireNonNull);
```

Comment:
since this is looping over all values in the list anyway, it shoul also check that each tag's type matches the provided type

Reply:
ah, i see that there's a pull request for this already (#81)
i guess it's easier to just add the check here and close #81

Reply:
Fixed :)